changelog/3960.removal.rst

```diff
@@ -0,0 +1 @@
+Add `deprecated` decorator to `sunpy.instr.aia.aiaprep` in preparation for removal in 3.1
```

Comment:
This should be user facing, so it should explain what's been deprecated, what the alternative is and when it's going away.